examples/parsed_view_func.py

```diff
@@ -0,0 +1,177 @@
+"""
+Example of auto load doc file by endpoint name(view function/method name),
+parsed by flask_restful.reqparse.RequestParser and validated by jsonschema
+"""
+import os
+
+from flask import Flask, request, jsonify, Response
+from flask.views import MethodView
+from flasgger import Swagger
+
+
+app = Flask(__name__)
+app.config['SWAGGER'] = {
+    'title': 'Flasgger ParsedMethodView Example',
```

Comment:
Regarding `ParsedMethodView` (three mentions, appears in UI) - on first seeing this in camel-case, I wondered if it was going to be a subclass of Flask `MethodView`, but it's not, it's just a name used in the example.  So maybe "Flasgger Parsed MethodView Example" might be better?